.github/pull_request_template.md

```diff
@@ -1,12 +1,18 @@
 ### What was wrong?
 
 Related to Issue #
+Closes #
 
 ### How was it fixed?
 
 ### Todo:
-- [ ] Add entry to the [release notes](https://github.com/ethereum/web3.py/blob/main/newsfragments/README.md)
+
+- [ ] Clean up commit history
+
```

Comment:
nit: I've been meaning to comment on this... can we keep no blank lines between these? It formats much nicer for a checkbox list like this one.  Thoughts?

Reply:
noooooooooproblem